src/meson.build

```diff
@@ -57,7 +57,7 @@ endif
 # target: test-*
 #
 
-test_api = executable('test-api', ['test-api.c'], link_with: libcrbtree_shared)
+test_api = executable('test-api', ['test-api.c'], dependencies: libcrbtree_dep)
```

Comment:
We want the API tests to link dynamically so we test the symbol visibility. This change would disable that test.